samples/src/Microsoft.Azure.IIoT.App/src/Connected Services/Application Insights/ConnectedService.json

```diff
@@ -0,0 +1,7 @@
+﻿{
```

Comment:
Is this file needed?  If not, please remove.

Reply:
I checked this file is simply a marker to the Visual Studio "Connected Services" tooling that Application Insights was installed. Removed